src/inmanta/plugins.py

```diff
@@ -21,40 +21,35 @@
 import inspect
 import os
 import subprocess
+import typing
 import warnings
-from collections import abc
-from typing import TYPE_CHECKING, Any, Callable, Literal, Optional, Type, TypeVar
 
+import inmanta.ast as ast
 import inmanta.ast.type as inmanta_type
-from inmanta import const, protocol, util
-from inmanta.ast import LocatableString, Location, Namespace, Range, RuntimeException, TypeNotFoundException, WithComment
-from inmanta.ast.type import NamedType
-from inmanta.config import Config
-from inmanta.execute.proxy import DynamicProxy
-from inmanta.execute.runtime import QueueScheduler, Resolver, ResultVariable
-from inmanta.execute.util import NoneValue, Unknown
-from inmanta.stable_api import stable_api
-from inmanta.warnings import InmantaWarning
+import inmanta.warnings
```

Comment:
This one can't be shortened, as we also import warnings: keep it like this or make it into inmanta_warnings?

Reply:
By making it into `inmanta_warnings` we don't win anything, right? In that case I would keep it like this.